speculos/mcu/seproxyhal.py

```diff
@@ -438,13 +443,26 @@ def handle_finger(self, x: int, y: int, pressed: bool):
         packet += x.to_bytes(2, 'big')
         packet += y.to_bytes(2, 'big')
 
-        self.packet_thread.queue_packet(SephTag.FINGER_EVENT, packet)
+        self.socket_helper.queue_packet(SephTag.FINGER_EVENT, packet)
+
+    def handle_ticker_request(self, action):
+        if action == "pause":
+            self.time_ticker_thread.pause()
+        elif action == "resume":
+            self.time_ticker_thread.resume()
+        elif action == "single-step":
+            self.time_ticker_thread.add_tick(wait_fully_processed=True)
 
     def handle_wait(self, delay: float):
         '''Wait for a specified delay, taking account real time seen by the app.'''
-        start = self.packet_thread.get_processed_ticks_count()
-        while (self.packet_thread.get_processed_ticks_count() - start) * TICKER_DELAY < delay:
-            time.sleep(TICKER_DELAY)
+        expected_ticks = int(delay / 0.1)
+        if not self.time_ticker_thread.paused:
+            start = self.socket_helper.get_processed_ticks_count()
```

Comment:
Method `get_processed_ticks_count()` is not in the scope of this PR, but at this point I was under the impression it was a method doing non-trivial things. It turns out it is a boilerplate for `self.socket_helper.ticks_count`, so I don't understand what we gain from calling a method, but IMHO we lose clarity and some performance.

I suggest either:
- to remove the calls to this function and use `self.socket_helper.ticks_count` directly.
- if we want to protect `ticks_count` from being written from outside of `SocketHelper`, to use a `@property` instead.

That way this time would be clearer, it's just comparing two `int`s, nothing is calculated.